tools/run_tests/run_xds_tests.py

```diff
@@ -190,7 +190,7 @@ def parse_port_range(port_arg):
                   default='global/networks/default',
                   help='GCP network to use')
 argp.add_argument('--service_port_range',
-                  default='8080:8110',
+                  default='8080:8280',
```

Comment:
optional: we can expand the port range to a much larger numbers. E.g., 1k? 50k?

Reply:
We can only use the ports allowed by the pre-configured firewall rule.

Reply:
Oh, TIL. In that case, looks good.